[Bug]: Can't reproduce tutorial - webpack issue

### Guidelines

- [X] I agree to follow this project's Contributing Guidelines.

### Project Version

0.2.3

### Platform and OS Version

_No response_

### Existing Issues

_No response_

### What happened?

While following tutorial "Porting Liquid Oxygen to Shiny", after `yarn webpack` command I get the following error:

```
ERROR in ./node_modules/@emdgroup-liquid/liquid/dist/react.js 4:0-61
Module not found: Error: Can't resolve './react-component-lib' in '.../js/node_modules/@emdgroup-liquid/liquid/dist'
Did you mean 'index.js'?
BREAKING CHANGE: The request './react-component-lib' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

### Steps to reproduce

1. Follow "Porting Liquid Oxygen to Shiny" [tutorial](https://appsilon.github.io/shiny.react/articles/shiny-react.html).
2. Error should appear after `yarn webpack` command.

### Expected behavior

Webpack compiles without error.

### Attachments

_No response_

### Screenshots or Videos

_No response_

### Additional Information

_No response_

--- a/vignettes/shiny-react.Rmd
+++ b/vignettes/shiny-react.Rmd
@@ -99,7 +99,7 @@ LdLoading <- component("LdLoading")
 In the `js` directory we use `yarn` to add the Liquid Oxygen library.
 ```sh
 yarn init --yes
-yarn add @emdgroup-liquid/liquid
+yarn add @emdgroup-liquid/liquid@3.0.0
 ```
 
 In order to use react components we need to find where package exports are defined first. We need to look for export keyword with names of components. In case of this package, exports can be found in `@emdgroup-liquid/liquid/dist/react`.

